STATUS: INLINE FAIL
MODIFIED: com.j256.ormlite.dao.RuntimeExceptionDaoTest.testStartThreadConnectionThrows
MODIFIED FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
CLEANER: com.j256.ormlite.logger.LoggerFactoryTest.testLogFactoryProperty
CLEANER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
POLLUTER: com.j256.ormlite.logger.LoggerFactoryTest.testSetLogFactory
POLLUTER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 19.437000036239624
==========================
@@ -1010,0 +1010,3 @@
+ {
+     LoggerFactory.setLogBackendFactory(null);
+ }
